README.md

```diff
@@ -42,7 +42,8 @@ PARAM_NAME=VALUE` commands.
 
 Depending on the nature and configuration of your S3-compatible
 service, some of these values may be required to set. For example, on
-AWS S3, you will need to set `S3_REGION` to a non-empty value. For a
-Ceph Object Gateway that doesn’t set
+AWS S3, you will need to set `S3_REGION` to a non-empty value. 
+If you want to use an alternative S3-compatible service, you need to set the 
+`S3_HOST` and `S3_PORT` parameters. For a Ceph Object Gateway that doesn’t set
```

Comment:
This sentence is becoming poorly readable. Make this a bullet list please, like so:

```markdown
For example:
* On AWS S3, you will need to set `S3_REGION` to a non-empty value. 
* If you want to use an alternative S3-compatible service, you need to set the 
  `S3_HOST` and `S3_PORT` parameters.
* For a Ceph Object Gateway that doesn’t set
  [rgw_dns_name](https://docs.ceph.com/en/latest/radosgw/config-ref/#confval-rgw_dns_name),
  you will need `S3_ADDRESSING_STYLE: path`.